.github/workflows/release.yml

```diff
@@ -41,6 +40,32 @@ jobs:
           python setup.py sdist bdist_wheel
           twine upload dist/*
 
```

Comment:
I am wondering if it is possible to use this github action for Merlin: https://github.com/caraml-dev/caraml-store/blob/main/.github/workflows/sdk_release.yaml#L34

Reply:
#TIL! Thanks for the suggestion. I'll take a look at this to see how viable it is.

Reply:
I don't get a change to take a look at this. Will try to pick this in the future PR.